docs/content/docs/dev/table/functions/udfs.md

```diff
@@ -227,6 +227,20 @@ env.from("MyTable").select(call(classOf[MyConcatFunction], $"a", $"b", $"c"));
 {{< /tab >}}
 {{< /tabs >}}
 
+{{< hint info >}}
+`TableEnvironment` provides two overload methods to create temporary system function with an `UserDefinedFunction`:
+
+- *createTemporarySystemFunction(
+  String name, Class<? extends UserDefinedFunction> functionClass)*
+- *createTemporarySystemFunction(String name, UserDefinedFunction functionInstance)*
+
+It is recommended to use `functionClass` over `functionInstance` as far as you can, 
```

Comment:
May be it would make sense to mention that in case of `functionClass` it requires the ability to invoke constructor with no args